lib/ansible/modules/database/mysql/mysql_replication.py

```diff
@@ -103,6 +103,13 @@
         required: false
         default: null
         version_added: "2.0"
+    master_use_gtid:
+        description:
+            - MariaDB Global Transaction ID parameter
+        type: str
+        choices: [ current_pos, 'no', slave_pos ]
```

Comment:
The use of 'no' is problematic in YAML since it must be quoted to be a string.
If it is not quoted it turns into a boolean False, which is not an accepted value according to `choices`.
So we may want to use a different value to disable this.

```yaml
    master_use_gtid: disabled
```

Reply:
Or (additionally) allow a falsy value to evaluate to `MASTER_USE_GTID=no`, but that would require changing the type to "string or bool"

Reply:
Impossible, the value is a string, so it will translate to 'False', which I'd rather not add to the `choices` list as well. Using `no` or `yes` as a string value is problematic, best to avoid.